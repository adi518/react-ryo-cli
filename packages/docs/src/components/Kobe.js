import styled from 'styled-components';

import kobe from 'images/kobe.png';

const Kobe = styled.img.attrs({ src: kobe, alt: 'R.I.P. Kobe Bryant' })`
  right: 0;
  bottom: 0;
  width: 10rem;
  opacity: 0.025;
  position: absolute;
  transform: translateX(-1rem);
  transition: width var(--common-transition-time),
    opacity var(--common-transition-time);

  &:hover {
    opacity: 1;
  }

  @media (max-width: 575.98px) {
    width: 6rem;
  }
`;

export default Kobe;
