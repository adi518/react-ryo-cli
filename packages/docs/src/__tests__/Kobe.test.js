import Kobe from 'components/Kobe';

describe('Kobe', () => {
  it('renders', () => {
    const wrapper = shallow(<Kobe />);
    expect(wrapper).toMatchSnapshot();
  });
});
