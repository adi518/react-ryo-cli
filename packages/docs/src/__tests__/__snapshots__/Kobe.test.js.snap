// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Kobe renders 1`] = `
<img
  alt="R.I.P. Kobe Bryant"
  className="sc-AxjAm hYQIsj"
  src="kobe.png"
/>
`;
